mentoring/mentoring.py

```diff
@@ -44,12 +44,18 @@
 log = logging.getLogger(__name__)
 loader = ResourceLoader(__name__)
 
+_default_theme_config = {
+    'package': 'mentoring',
+    'locations': ['public/themes/lms.css']
```

Comment:
Some styles like `.mentoring .checkmark-incorrect { color: }` are now specified in both `mentoring.css` and `themes/lms.css`. Perhaps we can remove the colors from mentoring.css? Or is it important to keep that as a fallback?